Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -120,18 +132,16 @@ extension LoginViewController: EndpointDelegate{
             }
             return
         }
-
+        
         DispatchQueue.main.async {
             //Make sure that the authorization error message is hidden
             self.unauthorizedLabel.isHidden = true
             //Extract the users name and will eventually call a segue to next screen.
-            self.currentUser.extractUser(fromData: unwrappedData[0])
```

Comment:
This line should be added back in and the call to self.parseCurrentUser should be removed.